src/lib.rs

```diff
@@ -1066,6 +1079,13 @@ bitflags! {
     }
 }
 
+bitflags! {
+    /// TODO
+    #[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
+    pub struct SignatureFlags: u32 {
+    }
+}
```

Comment:
According to the documentation (https://learn.microsoft.com/en-us/windows/win32/api/sspi/nf-sspi-makesignature), these flags are package-specific:

> Package-specific flags that indicate the quality of protection.

So, you can use `u32` for the `pfQOP` parameter in the `make/verify_signature` methods. If you want to define some NTLM-specific flags, it's better to do it in the `ntlm` module.